docs/docs/starters.md

```diff
@@ -2,45 +2,55 @@
 title: "Gatsby Starters"
 ---
 
-The Gatsby CLI tool lets you install “starters”. These are boilerplate Gatsby sites maintained by the community.
+The Gatsby CLI tool lets you install **starters**, which are boilerplate Gatsby sites maintained by the community and intended for jump-starting development quickly.
 
-`gatsby new` helps you start your project by cloning the boilerplate, installing dependencies, and clearing Git history.
+## Installing starters
 
-When creating a new site, you can optionally specify a starter to base your new site on, either with the `[URL]` of the `[GIT_USER_NAME/REPO]` e.g.
+To clone a boilerplate starter, install its dependencies, and clear Git history, execute the `gatsby new` command.
```

Comment:
I think the part of clearing the Git history could confuse people. I could see them asking "am I responsible for that?" Doesn't `gatsby new` take care of that for you?